StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandler.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 // Emitted when there is an error loading connect js
 class OnLoadErrorMessageHandler: OnSetterFunctionCalledMessageHandler<OnLoadErrorMessageHandler.Values> {
     struct Values: Codable, Equatable {
-        let error: ErrorValue
+        let error: ConnectLoadingError
         
-        struct ErrorValue: Codable, Equatable {
+        struct ConnectLoadingError: Codable, Equatable, Error {
```

Comment:
🚫 Blocker

This was renamed to `EmbeddedComponentError` in API review and should be moved into its own file + made public. Alternative is remove `Error` conformance from this PR and address decoding the JS error into a `EmbeddedComponentError` error in a separate PR.

I'm not certain if we want to necessarily make `EmbeddedComponentError` conform to `Codable` because I think it would mean that it has to have public conformance and would affect the public API. So we might want to keep this as an internal type that `EmbeddedComponentError` is constructed from.